packages/fluentui/react-northstar/src/components/Tree/TreeItem.tsx

```diff
@@ -326,11 +327,11 @@ export const TreeItem: ComponentWithAs<'div', TreeItemProps> & FluentComponentSt
     </ElementType>
   );
 
-  const elementWithRef = <Ref innerRef={ref}>{element}</Ref>;
+  const elementWithRef = <Ref innerRef={elementRef}>{element}</Ref>;
```

Comment:
We should address this in next round when we will remove usages of `Ref` internally